splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeSystemProcedures.java

```diff
@@ -69,4 +69,15 @@ public static void LOAD_REGIONS(String serverName, final ResultSet[] resultSets)
             resultSets[0] = ProcedureUtils.generateResult("Error", e.getLocalizedMessage());
         }
     }
+
+    public static void RESTART_OLAP_SERVER(final ResultSet[] resultSets) throws StandardException, SQLException {
```

Comment:
Please add a test, maybe in OlapServerIT, similar to the one already there but using this procedure.